sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/entities/event/EventShareDataBuilder.java

```diff
@@ -67,20 +64,18 @@ public void doBusinessValidation(SormasToSormasEventDto sormasToSormasEventDto)
 
 	@Override
 	public SormasToSormasEventPreview doBuildShareDataPreview(Event event, ShareRequestInfo requestInfo) {
-		Pseudonymizer pseudonymizer =
-			dataBuilderHelper.createPseudonymizer(requestInfo);
-
+		Pseudonymizer pseudonymizer = dataBuilderHelper.createPseudonymizer(requestInfo);
 		return getEventPreview(event, pseudonymizer);
 	}
 
-	private EventDto getEventDto(Event event, Pseudonymizer pseudonymizer) {
-		EventDto eventDto = eventFacade.convertToDto(event, pseudonymizer);
+	@Override
+	protected EventDto getDto(Event event, Pseudonymizer pseudonymizer) {
 
-		eventDto.setReportingUser(null);
```

Comment:
it is a bit confusing what the overall change wants to be
looking at other parts i would say we need this nulling of the reporting user
some minimal unit tests would shed light on the whole thing as the ticket you are referencing is also a bit lacking in description

Reply:
Hey, the nulling of the user is exactly what causes the transmission to fail. https://github.com/hzi-braunschweig/SORMAS-Project/pull/9696 already did this for contacts. I can create a unit test.

Reply:
tests added